### PR TITLE
Added emojies, lines diff checker, app name

### PR DESCRIPTION
**Pull request by marinakalyuzhnaya**
### Problem solved:

1) added emojies to highlight comment with red/green colour for diff
2) added possibility to show details only for changed lines , please note, that this will work only if you have base lcov file, if not previous code will work without changes
3) added possibility to add app-name (needed mostly for singe repo with diff siblings)

### CODE REVIEW

File: src/comment.js [hunk 1]
```diff
@@ -2,6 +2,11 @@ import { details, summary, b, fragment, table, tbody, tr, th } from "./html";
 import { percentage } from "./lcov";
 import { tabulate } from "./tabulate";
 
+const renderEmoji = pdiff => {
+	if (pdiff < 0) return "🔴";
+	if (pdiff > 0) return "🟢";
```
Comment (Weetbix) [inline src/comment.js]: Lets use `>= 0` here, so that we always have an emoji and that 0% shows as green :)

**Review by Weetbix — APPROVED** [review]
Looks great, saw it in action with your other PR

**Review by eeshdarthvader — APPROVED** [review]
Nice

**Review by janory — APPROVED** [review]
LGTM! 👍

**Review by Weetbix — APPROVED** [review]
Looks great again, I've seen it in action :)

File: action.yml [hunk 2]
```diff
@@ -17,6 +17,9 @@ inputs:
   lcov-base:
     description: The location of the lcov file for the base branch. Applicable for single repo setup.
     required: false
+  app-name:
+    description: The concrete app name to dispaly on comment
```
Comment (eeshdarthvader) [inline action.yml]:
```suggestion
    description: App name to display on comment
```

File: dist/main.js [hunk 3]
```diff
@@ -6024,6 +6024,28 @@ function uncovered(file, options) {
...
 }
 
+/**
+ * Compares two arrays of objects and returns with unique lines update
+ * @param {number} pdiff value from diff percentage
+ * @returns {string} emoji string for negative/positive pdiff
+ */
+const renderEmoji = pdiff => {
+    if (pdiff < 0) return "🔴";
+    if (pdiff >= 0) return "🟢";
```
Comment (eeshdarthvader) [inline dist/main.js]:
```suggestion
    if (pdiff < 0) return "🔴";
    return "🟢";
```

also it shows like this. For  >=0 change the emoji is weird.

Screenshot 2021-02-09 at 15 41 02

File: dist/main.js [hunk 4]
```diff
@@ -6327,6 +6366,7 @@ async function main() {
                   lcovBaseArrayForMonorepo,
                   options,
               ),
+				hiddenHeader: appName ? `<!-- ${appName}-jest-reporter-action -->` : `<!-- monorepo-jest-reporter-action -->`
```
Comment (eeshdarthvader) [inline dist/main.js]:
```suggestion
				hiddenHeader: appName ? `<!-- ${appName}-code-coverage-assistant-action -->` : `<!-- monorepo-code-coverage-assistant-action -->`
```

File: src/comment.js [hunk 5]
```diff
@@ -2,6 +2,28 @@ import { details, summary, b, fragment, table, tbody, tr, th } from "./html";
...
 import { tabulate } from "./tabulate";
 
+/**
+ * Compares two arrays of objects and returns with unique lines update
+ * @param {number} pdiff value from diff percentage
+ * @returns {string} emoji string for negative/positive pdiff
+ */
+const renderEmoji = pdiff => {
+    if (pdiff < 0) return "🔴";
+    if (pdiff >= 0) return "🟢";
```
Comment (eeshdarthvader) [inline src/comment.js]:
```suggestion
    if (pdiff < 0) return "🔴";
     return "🟢";
```

**Review by eeshdarthvader — CHANGES_REQUESTED** [review]
Looks good. Few changes added.

File: dist/main.js [hunk 6]
```diff
@@ -6024,6 +6024,28 @@ function uncovered(file, options) {
...
 }
 
+/**
+ * Compares two arrays of objects and returns with unique lines update
+ * @param {number} pdiff value from diff percentage
+ * @returns {string} emoji string for negative/positive pdiff
+ */
+const renderEmoji = pdiff => {
+    if (pdiff.toFixed(2) < 0) return "🔴";
+    return "🟢";
```
Comment (eeshdarthvader) [inline dist/main.js]: shouldn't this be green filled circle ?

**Comment by marinakalyuzhnaya** [comment]
> Looks great again, I've seen it in action :)

I already have some improvements ideas, how to show only changed lines without having a base file. :)

File: dist/main.js (hunk 6 above)
Comment (marinakalyuzhnaya) [reply dist/main.js]: I do not get the question

**Review by eeshdarthvader — APPROVED** [review]
Looks great. ❤️

**Pull request merged**